src/js/apps/patients/schedule/schedule_state.js

```diff
@@ -75,13 +78,13 @@ export default Backbone.Model.extend({
     const filtersState = this.getFilters();
     const clinicianId = this.get('clinicianId');
     const customFilters = omit(filtersState, 'groupId');
-    const status = [STATE_STATUS.QUEUED, STATE_STATUS.STARTED].join(',');
+    const state = this.states.filterByDone().pluck('id').join(',');
```

Comment:
I also no it's not a huge deal, but I think we're doing it enough places we should make a:
```js
getFilterIds() {
  return this.map('id').join(',');
}
```
on the state collection..

I dunno why I always default to `pluck`, but it's essentially an alias of `map` with no benefit.

Reply:
Added this in the last push I just made.